ci/jenkins/test.sh

```diff
@@ -495,13 +499,13 @@ function run_conformance_windows {
     export PATH=$GOROOT/bin:$PATH
     export KUBECONFIG=$KUBECONFIG_PATH
 
-    if [[ "$TESTCASE" == "windows-conformance" ]]; then
+    if [[ "$TESTCASE" =~ "windows-networkpolicy-temp-disabled" ]]; then
```

Comment:
You use "not" in the "if" statement, but says user wanna process in the content, is it as your design?

Reply:
I think `=~` means equal or include while `!=` is no equal.